### ISSUE TITLE
Tooltip outputs broken in 3.2.1

### ISSUE THREAD

**Issue by Saxayone** [WeakAuras/WeakAuras2#2979]
## Describe the bug
Tooltip output is currently always blank. Both using default %tooltip, %tooltip1 outputs and custom outputs of aura_env.states[1].tooltip, aura_env.states[1].tooltip1

**Do you have an error log of what happened?**
None

### To Reproduce
1.Make any weakaura that tracks a buff
2. Enable "Use tooltip information"
3. Set output text to use the tooltip and it will show blank.

Example: https://wago.io/ybfrklrM0

**Screenshots**
https://i.snipboard.io/7LR251.jpg - Trigger properties
https://i.snipboard.io/Ug7hG3.jpg - Text properties
https://i.snipboard.io/2YCQD3.jpg - Blank output

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes

**Which version of WeakAuras are you using?**
3.2.1

**Are you on World of Warcraft Classic or Retail?**
- [ x ] Retail

**Was it working in a previous version? If yes, which was the last good one?**
Yes, I think pre 3.2.0

### Additional Info
none

**Comment by InfusOnWoW** [comment]
I don't have any idea what that buff id is, but replacing it with any id shows that he aura works correctly. If there was a problem with tooltips, we would get plenty of bug reports. And there's certainly no reason at all to suspect that this buff would be different.

**Comment by Saxayone** [comment]
It seemed to be linked to this error.
https://github.com/WeakAuras/WeakAuras2/issues/2966

**Comment by InfusOnWoW** [comment]
But that has been already fixed.

**Comment by Saxayone** [comment]
Yeah sorry, was posting mainly for posterity's sake and should have explained a bit better.

On the pre-fix version, I had noticed after this was closed that the linked error happen which then seemed to have been the root cause that issue I had.

Any time after the linked error had occurred even once then, %tooltip, %tooltip1 outputs and custom outputs of aura_env.states[1].tooltip, aura_env.states[1].tooltip1 would break until relog.

It was fixed with the linked issue being fixed.
